FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Minimal_Windows_Simulator/DemoTasks/SimpleUDPClientAndServer.c

```diff
@@ -329,10 +344,12 @@ Socket_t xListeningSocket;
 	after the network is up, so the IP address is valid here. */
 #if defined( ipconfigIPv4_BACKWARD_COMPATIBLE ) && ( ipconfigIPv4_BACKWARD_COMPATIBLE == 0 )
 	FreeRTOS_GetEndPointConfiguration( &ulIPAddress, NULL, NULL, NULL, pxNetworkEndPoints );
+	xBindAddress.sin_address.ulIP_IPv4 = ulIPAddress;
 #else
 	FreeRTOS_GetAddressConfiguration( &ulIPAddress, NULL, NULL, NULL );
-#endif
 	xBindAddress.sin_addr = ulIPAddress;
+#endif

```

Comment:
Let us follow the same the format for #endif in all the places. Some places have /* defined( ipconfigIPv4_BACKWARD_COMPATIBLE ) && ( ipconfigIPv4_BACKWARD_COMPATIBLE == 0 ) */ and some do not.